packages/web/src/components/upload/TrackPreview.module.css

```diff
@@ -116,13 +117,27 @@
   flex: 1 1 0;
 }
 
+.sizeContainer {
+  position: relative;
+  width: fit-content;
+}
+
 .fileSizeText {
-  display: flex;
+  opacity: 1;
+  transition: var(--hover);
+}
+
+.removeButtonContainer {
```

Comment:
i dont have the full context of this experience, but do we want to use absolute positioning?

Reply:
yea previously i was instantly switching between the file size text and the delete icon, but in order to fade i need them both to exist and just fade the opacities. using absolute positioning here to have them stack on top of each other so they appear in the same spot

Reply:
ah, gotcha! thanks that makes sense